doc/engineering-reference/src/loop-equipment-sizing-and-other-design-data/component-sizing.tex

```diff
@@ -968,8 +1019,10 @@ \subsubsection{System Coils}\label{system-coils-5}
 
 \emph{CapModFac} = \emph{CurveValue}(CCapFTemp,\emph{T\(_{mix,wb}\)},\emph{T\(_{outside}\)})
 
+\emph{Q\(_{fan,heat,des}\)} = design fan heat (W) - see Section \ref{design-fan-heat}
```

Comment:
How do I get the \dot{Q} inside the \emph{}. I tried a few different ways and nothing worked.